tests/test_pretest.py

```diff
@@ -257,3 +259,24 @@ def test_stop_pfcwd(duthosts, enum_dut_hostname, tbinfo):
     Please add public pretest above this comment and keep internal
     pretests below this comment.
 """
+
+def prepare_autonegtest_params(duthosts, fanouthosts):
+    from tests.platform_tests.link_flap.link_flap_utils import build_test_candidates
```

Comment:
I don't think it is a good practice to import something from other tests. This will introduce strong dependency between tests. If something went wrong, it would be very difficult to debug. 
If something is shared by multiple tests, suggest moving it to `tests/common`.